spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -1073,6 +1281,12 @@ def _get_data(self, vertex_slice):
 
         return numpy.concatenate(data)
 
+    def set_max_rewires_per_ts(self, max_rewires_per_ts):
+        """
+        :param int max_rewires_per_ts: the maximum rewires per timestep
+        """
+        self.__max_rewires_per_ts = max_rewires_per_ts
+
```

Comment:
If this is going to be max_events_per_ts, this would need to take a variable name too, and should be gated to ensure it isn't called against other variable types.